src/api.ts

```diff
@@ -0,0 +1,26 @@
+import * as rp from 'request-promise-native';
+import * as url from 'url';
+
+import { configuration } from './configuration';
+import { Flag } from './models';
+
+// LaunchDarklyAPI is a wrapper around request-promise-native for requesting data from LaunchDarkly's REST API. The caller is expected to catch all exceptions.
```

Comment:
This should eventually be replaced by one of our fancy api clients, but just setting this up as scaffolding to make other work easier. Behavior is very similar to the old `getFeatureFlag` function in `flags.ts`